cms/djangoapps/modulestore_migrator/rest_api/v1/views.py

```diff
@@ -571,31 +573,63 @@ def get(self, request: Request):
         """
         Handle the migration info `GET` request
         """
-        source_key = request.query_params.get("source_key")
-        target_key = request.query_params.get("target_key")
-        target_collection_key = request.query_params.get("target_collection_key")
-        task_uuid = request.query_params.get("task_uuid")
-        is_failed: str | bool | None = request.query_params.get("is_failed")
-        if not target_key:
+        target_key: LibraryLocatorV2 | None
```

Comment:
Again, this REST API handler has no functional changes; it's just being stricter with validation so that client errors become 400s instead of 500s.